src/components/timeCard.vue

```diff
@@ -5,72 +5,34 @@
       <div class="time-row">
         <div class="time-checkbox">
           <select
-            v-model="startTime"
+            v-model="addTime.startTime"
             class="m-md-2 select"
             id="startTime"
             required
           >
-            <option selected disabled>Choose a time</option>
-            <option value=""></option>
-            <option>00:00</option>
-            <option>01:00</option>
-            <option>02:00</option>
-            <option>03:00</option>
-            <option>04:00</option>
-            <option>05:00</option>
-            <option>06:00</option>
-            <option>07:00</option>
-            <option>08:00</option>
-            <option>09:00</option>
-            <option>10:00</option>
-            <option>11:00</option>
-            <option>12:00</option>
-            <option>13:00</option>
-            <option>14:00</option>
-            <option>15:00</option>
-            <option>16:00</option>
-            <option>17:00</option>
-            <option>18:00</option>
-            <option>19:00</option>
-            <option>20:00</option>
-            <option>21:00</option>
-            <option>22:00</option>
-            <option>23:00</option>
+            <option
+              v-for="time in timeList"
+              :key="time.id"
+              :value="{ value: time.value, label: time.label }"
```

Comment:
larka tumhain yahan ab label ki i zarrorat ni simple value ki ha. ku label tum baad main abb use to nahi kar raha

Reply:
is label ko neechay disabled ke property may use kea hai
kyu kay wo value ko consider nae kar raha hai.
or agar yahan say label get nae karta to neechay disabled function may label ke value get nae hoti